README.md

```diff
@@ -291,15 +291,15 @@ For `linux` you need Docker running, on Mac or Linux. For `macos`, you need a Ma
 
 Optional.
 
-Space-separated list of builds to build and skip. Each build has an identifier like `cp27-manylinux1_x86_64` or `cp34-macosx_10_6_intel` - you can list specific ones to build and `cibuildwheel` will only build those, and/or list ones to skip and `cibuildwheel` won't try to build them.
+Space-separated list of builds to build and skip. Each build has an identifier like `cp27-manylinux_x86_64` or `cp34-macosx_10_6_intel` - you can list specific ones to build and `cibuildwheel` will only build those, and/or list ones to skip and `cibuildwheel` won't try to build them.
 
 When both options are specified, both conditions are applied and only builds with a tag that matches `CIBW_BUILD` and does not match `CIBW_SKIP` will be built.
 
 The format is `python_tag-platform_tag`. The tags are as defined in [PEP 0425](https://www.python.org/dev/peps/pep-0425/#details).
```

Comment:
```suggestion
The format is `python_tag-platform_tag`.
```

Our tweak from `manylinux2010` to `manylinux` means we've deviated from this spec, so let's remove the link.

Reply:
Ah, yes :-/ That's the downside of this PR, of course :-(

Maybe I'll still add something on using `--print-build-identifiers` to get the list.

Reply:
I tried to rephrase and keep the reference (since it still seems relevant for users of `cibuildwheel` to know about this PEP).

Just a suggestions. If you don't agree and want to completely get rid of the sentence, do just push that change before you merge! :)